core/Cargo.toml

```
@@ -16,17 +16,21 @@
 # under the License.
 
 [package]
-name = "comet"
+name = "datafusion-comet"
 version = "0.1.0"
+homepage = "https://datafusion.apache.org/comet"
+repository = "https://github.com/apache/datafusion-comet"
+authors = ["Apache DataFusion <dev@datafusion.apache.org>"]
+description = "Apache DataFusion Comet: Spark native engine"
```

```suggestion
description = "Apache DataFusion Comet: Apache Spark native query execution plugin"
```

This will be consistent with what in DataFusion known userlist